modules/contrib/cyber_bridge/BUILD

```diff
@@ -0,0 +1,27 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "cyber_bridge",
+    srcs = [
+        "bridge.cc",
+        "node.cc",
+        "node.h",
+        "client.cc",
+        "client.h",
+        "clients.cc",
+        "clients.h",
+        "server.cc",
+        "server.h",
```

Comment:
Prefer slim targets. Each pair of `.cc` and `.h` should be in its own cc_library or cc_binary, and claims dependency relationship clearly.

Reply:
I split out two libraries: bridge_client and bridge_server.
There are circular inclusion in node.cc, client.cc, and clients.cc. So I put them in one library.